ports/rp2/pendsv.h

```diff
@@ -27,6 +27,7 @@
 #define MICROPY_INCLUDED_RP2_PENDSV_H
 
 #include <stddef.h>
+#include "hardware/irq.h"
```

Comment:
I don't think this include is needed (or maybe needs to be moved to `pendsv.c`).

Reply:
> This header has `#define IRQ_PRI_PENDSV PICO_LOWEST_IRQ_PRIORITY` in it, so I added this on the basis of "include what you use".

Ah, I see.  That makes sense.

> But I now realise the macro define could also be moved to pendsv.c, as (as of this PR) this is the only file which uses it.

Yes indeed it could be moved.  I'm happy either way (move or keep here).

Reply:
Have moved!